desktop/apps/consignments2/templates/landing.jade

```diff
@@ -5,9 +5,11 @@ block head
 
```

Comment:
Sorta confused by the jade file...so are all the React components just purely components that are stil part of the page/layout (implemented in jade)? Or do they interact/render jade?